sql/sql_const.h

```diff
@@ -29,7 +29,7 @@
 /* extra 4+4 bytes for slave tmp tables */
 #define MAX_DBKEY_LENGTH (NAME_LEN*2+1+1+4+4)
 #define MAX_ALIAS_NAME 256
-#define MAX_FIELD_NAME 34			/* Max colum name length +2 */
+#define MAX_FIELD_NAME (NAME_LEN+1)			/* Max colum name length +1 */
```

Comment:
Oh, i see what was the problem in markup now: there are tabs used in between. So:
1. Note that tabs are configured at width 8 for mariadb.
2. Please use spaces here instead. The comment should be aligned with other neghbouring comments